### PR TITLE
Feature: Add group page tabs

### PR DESCRIPTION
**Pull request by justinehell**
In Group Page :
- Add Groups tab
     - Attach/detach group -> to check detach method and params order
- Add Permissions tab
     - No action

Bonus
In Role Page : 
- In Role tab
     - Attach/detach role

### CODE REVIEW

File: src/services/GroupService.js [hunk 1]
```diff
@@ -94,3 +105,31 @@ export async function dissociateGroupAndUser(userLogin, groupId) {
...
+  });
+}
+
+/**
+ * Dissociate group and group.
+ * @param {string} groupIdToDetach - Group id to detach to the group.
+ * @param {string} groupId - Group id.
+ * @returns {Promise<object>} Promise with nothing on success otherwise an error.
+ */
+export async function dissociateGroupAndGroup(groupIdToDetach, groupId) {
```
Comment (Zorin95670) [inline src/services/GroupService.js:131]: invert groupIdToDetach, groupId

File: src/components/dialog/DetachGroupFromGroupDialog.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+/**
+ * Detach group, send event to reload groups and close dialog.
+ * @returns {Promise<void>} Promise with nothing on success.
+ */
+async function onSubmit() {
+  submitting.value = true;
+
+  // TODO: generic error management
+  await GroupService.dissociateGroupAndGroup(group.value.id, grouptoDetach.value.id);
```
Comment (Zorin95670) [inline src/components/dialog/DetachGroupFromGroupDialog.vue:65]: you may have to invert here too

File: src/components/dialog/AttachRoleToGroupDialog.vue [hunk 3]
```diff
@@ -76,7 +76,9 @@ const roles = ref([]);
  * @returns {Promise<void>} Promise with nothing on success.
  */
 async function search() {
-  return RoleService.find().then((data) => {
+  return RoleService.find({
+    name: 'not_SUPER_ADMINISTRATOR',
```
Comment (Zorin95670) [inline src/components/dialog/AttachRoleToGroupDialog.vue:80]: add:
```
id: `not_${currentRole.id}`,
```

File: tests/e2e/support/step_definitions/api.js [hunk 4]
```diff
@@ -291,6 +351,16 @@ function setRoleIntercepts() {
     },
   });
 
+  cy.intercept('GET', '/api/roles?name=not_SUPER_ADMINISTRATOR', {
```
Comment (Zorin95670) [inline tests/e2e/support/step_definitions/api.js:354]: don't forget to add intercept for
`/api/roles?id=not_X&name=not_SUPER_ADMINISTRATOR`

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: src/services/GroupService.js (hunk 1 above)
Comment (justinehell) [reply src/services/GroupService.js:131]: done

File: src/components/dialog/AttachRoleToGroupDialog.vue (hunk 3 above)
Comment (justinehell) [reply src/components/dialog/AttachRoleToGroupDialog.vue:80]: done but not here

File: tests/e2e/support/step_definitions/api.js (hunk 4 above)
Comment (justinehell) [reply tests/e2e/support/step_definitions/api.js:385]: done

**Pull request merged**